src/util/aquarius.ts

```diff
@@ -177,3 +215,44 @@ export async function getPopularTags(size = 10) {
     console.error(error)
   }
 }
+
+export async function getAllTags() {
+  try {
+    const tagQuery = {
+      ...getBaseQuery(),
+      aggs: {
+        popular_tags: {
+          significant_text: {
+            field: 'service.attributes.additionalInformation.tags'
+          }
+        }
+      },
+      size: 0
+    }
+
+    const response = await axios.post(apiBasePath, tagQuery)
+
+    return response.data
+  } catch (error) {
+    console.error(error)
```

Comment:
thought(non-blocking): is there some toast notification that we could utilize here instead of erroring using console?